api/src/controllers/users.ts

```diff
@@ -30,7 +33,9 @@ export default class UsersController {
         if (duplicate) {
             return res.status(409).json({ msg: "User already exists" });
         }
-        const hashedPassword = await bcrypt.hash(password, 10);
+
+        let pass = password.toString();
+        const hashedPassword = await bcrypt.hash(pass, 10);
```

Comment:
Before developing I made some test cases in postman, it all had integer passwords, so instead of fixing the tests I did this casting. I can remove later if this bothers someone, but I also could it be like this because it don't really changes anything.